modules/ozoneBidAdapter.js

```diff
@@ -140,14 +141,13 @@ export const spec = {
 
     if (bidderRequest && bidderRequest.gdprConsent) {
       utils.logInfo('OZONE: ADDING GDPR info');
-      ozoneRequest.regs = {};
-      ozoneRequest.regs.ext = {}; // setting default values in case there is no additional information, like for the Mirror
-      ozoneRequest.regs.ext.gdpr = bidderRequest.gdprConsent.gdprApplies ? 1 : 0;
+      let apiVersion = utils.deepAccess(bidderRequest.gdprConsent, 'apiVersion', '1');
```

Comment:
At this point your above if statement assured `bidderRequest.gdprConsent` so instead of using deepAccess with a single notation apiVersion, a simple:

```
let apiVersion = bidderRequest.gdprConsent.apiVersion || '1';
```

May be a little cleaner.

Reply:
thanks have noted this to be updated/cleaned in our next update - assume it will be okay to leave as is for now (keen to get this version out there).